jerry-core/ecma/base/ecma-helpers.c

```diff
@@ -892,21 +968,28 @@ ecma_delete_property (ecma_object_t *object_p, /**< object */
 
     ecma_property_pair_t *prop_pair_p = (ecma_property_pair_t *) cur_prop_p;
 
-    for (int i = 0; i < ECMA_PROPERTY_PAIR_ITEM_COUNT; i++)
+    for (uint32_t i = 0; i < ECMA_PROPERTY_PAIR_ITEM_COUNT; i++)
```

Comment:
Why has this been changed to `uint _32`? I get that it may be better unsigned, but that's `unsigned int`. Why is the exact bit length needed?